### PR TITLE
Fix Overflow Fade In Stats Page "Post Published" Overflow

### LINKED ISSUE #34365: Stats Chart Tabs: "Post Published" Gradient
#### Steps to reproduce
1. Starting at URL: /stats/day in the Nightfall scheme 
2. Hover over a stat bar

Note the blue at the end of the post title, I expected that fade to match the other colours better.

Screenshot 2019-06-29 at 08 02 59

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Use color scheme in post title fade on stats page

#### Testing instructions
1. Navigate to `/stats/day` on WordPress.com with a more unconventional color scheme set ( Nightfall, Sakura, etc. )
2. Confirm the fade on the posts titles when mousing over a day in the bar chart with posts are a WordPress-y blue
Screen Shot 2019-09-13 at 11 48 13 AM
3. Repeat steps one and two on the branch
4. Confirm the title now blends into the background as it overflows Screen Shot 2019-09-13 at 2 10 19 PM

Fixes #34365

### CODE REVIEW

File: client/components/chart/style.scss [hunk 1]
```diff
@@ -434,18 +434,11 @@ $y-axis-padding: 0 20px 0 10px;
 			min-width: 0;
 
 			&::before {
-				content: '';
+				@include long-content-fade( $color: var( --color-primary-rgb ) );
```
Comment (mmtr) [inline client/components/chart/style.scss]: What if we use here the same color used for the [background](https://github.com/Automattic/wp-calypso/blob/fea949a8e688300e1dde2b51b8dbb451bf1721f4/client/components/chart/style.scss#L111)?

```suggestion
				@include long-content-fade( $color: var( --color-neutral-600-rgb ) );
```

That way it'll fade to the background of the tooltip, which looks a bit nicer IMO:

Screen Shot 2019-09-13 at 12 33 31

Comment (Aurorum) [reply client/components/chart/style.scss]: I prefer this approach too

Comment (cbauerman) [reply client/components/chart/style.scss]: Screen Shot 2019-09-13 at 2 10 19 PM

Also looks much, much better when the post title does not overflow 👍

File: client/components/chart/style.scss [hunk 2]
```diff
@@ -434,18 +434,11 @@ $y-axis-padding: 0 20px 0 10px;
 			min-width: 0;
 
 			&::before {
-				content: '';
+				@include long-content-fade( $color: var( --color-neutral-600-rgb ) );
```
Comment (mmtr) [inline client/components/chart/style.scss]: Seems that after https://github.com/Automattic/wp-calypso/pull/35949 the var name has changed:

```suggestion
				@include long-content-fade( $color: var( --color-neutral-60-rgb ) );
```

**Pull request merged**